Misc/NEWS.d/next/Library/2018-02-13-10-02-54.bpo-32839.McbVz3.rst

```diff
@@ -0,0 +1 @@
+Add after_info to tkinter.
```

Comment:
I think the news entry could/should be a bit more descriptive.

Reply:
If this were reopened, I would be willing to expand this.

Reply:
Concur.